### ISSUE TITLE
[Bug] It crashes when I lock my computer

### ISSUE THREAD

**Issue by skyf0l** [pit-ray/win-vind#252]
# About Bug

It's all in the title, win-vind crash when locking my computer.

# Environments

I installed the Portable version, default configuration

```
========== System Infomation ==========
[Windows]
      Edition: Windows 10 Enterprise
      Version: 22H2
Build Numbers: 10.0.22621
 Architecture: x64

[win-vind]
      Version: 5.13.0.0
```

Btw system informations showed up by win-vind are not up to date, it's a `Windows 11 Entreprise`, 22H2, build 22621.3593.

# Logs

I only have these two types of logs:

```
[Message] Version: huge
[Error] Access is denied.
[Error] Failed the tray update (int __cdecl WinMain(struct HINSTANCE__ *,struct HINSTANCE__ *,char *,int))
```

```
[Error] Access is denied.
[Error] Failed the tray update (int __cdecl WinMain(struct HINSTANCE__ *,struct HINSTANCE__ *,char *,int))
[Error] Access is denied.
[Error] An runtime exception occurred: failed sending keyboard event (void __cdecl vind::core::InputGate::uninstall_hook(void) noexcept)
[Error] Access is denied.
[Error] An runtime exception occurred: failed sending keyboard event (void __cdecl vind::core::InputGate::uninstall_hook(void) noexcept)
[Error] Access is denied.
[Error] An runtime exception occurred: failed sending keyboard event (void __cdecl vind::core::InputGate::uninstall_hook(void) noexcept)
```

**Comment by pit-ray** [comment]
Thanks for the bug report.
I was able to reproduce the crash caused by the lock.

Perhaps this is the cause of #196, I will fix it as a top priority.

> Btw system informations showed up by win-vind are not up to date, it's a Windows 11 Entreprise, 22H2, build 22621.3593.
Windows 11 is indicated by the same identifier as 10 on the API, so there is no problem.

Thanks.

**Comment by pit-ray** [comment]
@skyf0l 
I fixed this issue.
The corrected build is attached below so please try it out.
I will merge the changes after testing.
[win-vind_fix-252_64bit_portable.zip](https://github.com/user-attachments/files/15822281/win-vind_fix-252_64bit_portable.zip)

**Comment by skyf0l** [comment]
Perfect @pit-ray, it works!!

**Comment by pit-ray** [comment]
@skyf0l
I released this change as v5.13.1.
Thanks for your report.
